src/components/Banner/Banner.tsx

```diff
@@ -47,14 +51,41 @@ export interface BannerProps {
   stopAnnouncements?: boolean;
 }
 
-export class Banner extends React.PureComponent<BannerProps, never> {
+export class Banner extends React.PureComponent<BannerProps, State> {
+  state: State = {
+    showFocus: false,
+  };
+
   private wrapper = React.createRef<HTMLDivElement>();
 
   public focus() {
     this.wrapper.current && this.wrapper.current.focus();
+    this.setState({showFocus: true});
   }
 
   render() {
+    const {showFocus} = this.state;
+
+    const handleKeyUp = (evt: React.KeyboardEvent<HTMLDivElement>) => {
+      if (evt.target === this.wrapper.current) {
+        this.setState({showFocus: true});
+      } else if (showFocus === true) {
+        this.setState({showFocus: false});
```

Comment:
While tophatting I couldn't actually end up in this condition, which I think makes sense since, handleBlur looks like it's working. Can we 🔥the else if here?

Reply:
good catch. I thought it was needed when programmatically focused but blur handles that.